### ISSUE TITLE
Experimental false detection due to junk headers like "A few Missing Fills.[03/47] 

### ISSUE THREAD

**Issue by pipieye** [evilhero/mylar#2348]
**Describe the bug**
When poster post like the following:
A few Missing Fills.[03/47] - "Age of Conan - Valeria 001 (2019) (digital) (NeverAngel-Empire).cbr" yEnc
parser is detecting the 03 47 to be the issue number (below logs)

**Screenshots/Logs**
13-Sep-2019 17:23:13 - DEBUG :: mylar.NZB_SEARCH.1115 : SEARCH-QUEUE : parsed_info: {'series_volume': None, 'booktype': 'issue', 'issueid': None, 'sub': None, 'comiclocation': None, 'reading_order': None, 'parse_status': 'success', 'dynamic_name': u'AfewMissingFills03', 'series_name': u'A few Missing Fills 03', 'comicfilename': u'A few Missing Fills.[03/47] -  (2019)', 'issue_number': u'47', 'issue_year': '2019', 'scangroup': None, 'alt_series': u'A few Missing Fills 03 47', 'alt_issue': None, 'series_name_decoded': 'A few Missing Fills 03'}
13-Sep-2019 17:23:13 - DEBUG :: mylar.NZB_SEARCH.1124 : SEARCH-QUEUE : match_check: {'series_volume': None, 'issue_year': '2019', 'comiclocation': None, 'series_name': u'A few Missing Fills 03', 'scangroup': None, 'alt_series': u'A few Missing Fills 03 47', 'issueid': None, 'sub': None, 'booktype': 'issue', 'comicfilename': u'A few Missing Fills.[03/47] -  (2019)', 'issue_number': u'47', 'process_status': 'fail', 'alt_issue': None}
13-Sep-2019 17:23:13 - DEBUG :: mylar.NZB_SEARCH.1184 : SEARCH-QUEUE : Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
13-Sep-2019 17:23:13 - INFO :: mylar.search_init.395 : SEARCH-QUEUE : Could not find Issue 1 of Age of Conan: Valeria (2019) using experimental [api]

**Environment (please complete the following information):**
 - OS: MacOS
 - Branch / commit: Development/build 3fa8f08e64d4d35e202e042b424cd5110080cb21.

**Comment by ivtechboyinpa** [comment]
Have you encountered this on any other comics?

**Comment by evilhero** [comment]
@pipieye , if you can  - can you edit the findcomicfeed.py file located in the mylar subdirectory ? On line 146 you'll see:
```except_list=['releases', 'gold line', 'distribution', '0-day', '0 day', '0day', 'o-day', 're:']```

can you add ```'missing fills``` to the end of that list, so it looks like:
```except_list=['releases', 'gold line', 'distribution', '0-day', '0 day', '0day', 'o-day', 're:', 'missing fills']```

then save the file and start up Mylar and that should fix the problem. Please let me know for sure, so I can push the fix out once you've confirmed it's working on your end.

**Comment by pipieye** [comment]
i added the above and will test when another similar case comes up. However I found another glitch in experimental, not sure this is due to some recent change. Issue match to 012 then it says 12000?? So it did not grab the NZB.
----
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1098 : SEARCH-QUEUE : Entry: Superior Spider-Man 012 (2019) (Digital) (Zone-Empire).cbr
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.287 : SEARCH-QUEUE : Scanner group tag discovered: Zone-Empire
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.394 : SEARCH-QUEUE : NEWLY SPLIT REORGD: [u'Superior', u'Spider-Man', u'012', u'(2019)', u'(Digital)']
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.429 : SEARCH-QUEUE : checking date : 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.432 : SEARCH-QUEUE : date: 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.762 : SEARCH-QUEUE : datecheck: [{'date': u'2019', 'position': 3, 'mod_position': 24}]
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.768 : SEARCH-QUEUE : sctd: 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.777 : SEARCH-QUEUE : year verified as : 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.783 : SEARCH-QUEUE : year verified as: 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.815 : SEARCH-QUEUE : highest_series_position: 3
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.820 : SEARCH-QUEUE : possible_issuenumbers: [{'position': 2, 'validcountchk': False, 'number': u'012', 'mod_position': 20}]
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.826 : SEARCH-QUEUE : hyphen located at position: 15
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.828 : SEARCH-QUEUE : yearposition: 3
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.868 : SEARCH-QUEUE : issue number :012
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.897 : SEARCH-QUEUE : issue_position: 2
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.914 : SEARCH-QUEUE : issue verified as : 012
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.1041 : SEARCH-QUEUE : sf_highest_series_pos: [u'Superior', u'Spider-Man']
21-Nov-2019 19:20:02 - DEBUG :: mylar.parseit.1086 : SEARCH-QUEUE : series title possibly: Superior Spider-Man
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1115 : SEARCH-QUEUE : parsed_info: {'series_volume': None, 'booktype': 'issue', 'issueid': None, 'sub': None, 'comiclocation': None, 'reading_order': None, 'parse_status': 'success', 'dynamic_name': u'SuperiorSpider|Man', 'series_name': u'Superior Spider-Man', 'comicfilename': u'Superior Spider-Man 012 (2019) (Digital) (Zone-Empire).cbr', 'issue_number': u'012', 'issue_year': '2019', 'scangroup': u'Zone-Empire', 'alt_series': None, 'alt_issue': None, 'series_name_decoded': 'Superior Spider-Man'}
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1124 : SEARCH-QUEUE : match_check: {'justthedigits': u'012', 'issueid': None, 'match_type': None, 'sub': None, 'comiclocation': None, 'booktype': 'issue', 'series_name': u'Superior Spider-Man', 'comicfilename': u'Superior Spider-Man 012 (2019) (Digital) (Zone-Empire).cbr', 'annual_comicid': None, 'issue_year': '2019', 'volume': None, 'process_status': 'match', 'scangroup': u'Zone-Empire', 'alt_series': None, 'alt_issue': None, 'series_volume': None}
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1190 : SEARCH-QUEUE : year detected: 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1192 : SEARCH-QUEUE : year looking for: 2019
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1194 : SEARCH-QUEUE : 2019 - right years match baby!
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1380 : SEARCH-QUEUE : issue we found for is : 012
21-Nov-2019 19:20:02 - DEBUG :: mylar.NZB_SEARCH.1382 : SEARCH-QUEUE : integer value of issue we have found : 12000
